.gitignore

```diff
@@ -35,6 +35,9 @@ FirebaseSegmentation/Tests/Sample/GoogleService-Info.plist
 # FirebaseMLModelDownloader integration tests GoogleService-Info.plist
 FirebaseMLModelDownloader/Tests/Integration/Resources/GoogleService-Info.plist
 
+# FirebasePerformance dev test App and integration tests GoogleService-Info.plist
+FirebasePerformance/**/GoogleService-Info.plist
```

Comment:
Should it be in another PR?

Reply:
Yes. This is gone now after I merged it with master.